### ISSUE TITLE
Custom Data

### ISSUE THREAD

**Issue by huey0528** [XPandora/PhysGaussian#14]
Hi Tianyi 
Thank you for your great work. I am trying to apply it with some custom data which training from gaussian-splatting, but it is hard to follow your Data Preprocessing step by step. Could you mind provide more details on data preprocessing steps, eg how to use houdini for these steps. 
Meanwhile, may I know whether should I modify the config json file manually, or it can automally generated by any houdini？

Best Wishes

**Comment by XPandora** [comment]
Hi,

Thanks for your interest in our work.

I'll update the documentation with more instructions. For the config file, you should create your own manually. Houdini is used to view and check the distributions (positions) of Gaussian kernels to make sure you choose the correct parameters.

Best,
Tianyi
